reV/config/supply_curve_configs.py

```diff
@@ -293,6 +301,13 @@ def downwind(self):
         """Get the flag to exclude downwind as well as upwind sites"""
         return self.get('downwind', False)
 
+    @property
+    def offshore_wind_dirs(self):
+        """
+        Get flag to execute competitive wind directions for offshore sites
```

Comment:
Is this really a flag? Or was this docstring not updated? "dirs" seems to indicate it should be a points to wind direction data? Does it also work as a flag since the default is False?

After reviewing the CLI, it looks like this really is a flag. Maybe we can rename it to differentiate from the wind_dirs input?

Reply:
@grantbuster, good point, I was trying to make it less ambiguous than just offshore, but I guess I failed. Three thoughts:
1) wind_dirs_offshore
2) include_offshore_wind_dirs
3) include_wind_dirs_offshore